mkosi/__init__.py

```diff
@@ -6900,6 +6918,8 @@ def make_output_dir(config: MkosiConfig) -> None:
         return
 
     config.output_dir.mkdir(mode=0o755, exist_ok=True)
+    if config.map_permissions:
+        chown_to_running_user(config.output_dir)
```

Comment:
Pretty sure we can do this unconditionally (here and elsewhere) regardless of whether the option is set or not no? There wouldn't be too much benefit when --map-permissions is not used but it doesn't hurt either right?